sillsdev/web-languageforge
# Remove 'semantic domains only' interface languages

Fixes #1616

## Description

We are removing the languages that were listed as "semantic domains only" in the interface language picker in Project Settings.

### Type of Change

Only keep lines below that describe this change, then delete the rest.

- Bug fix (non-breaking change which fixes an issue)
- UI change

## Screenshots
Before:
![Screenshot (3)](https://user-images.githubusercontent.com/56163492/204706226-ffcc9fe5-34a1-42b3-8169-8771facfaada.png)

After:
![Screenshot (2)](https://user-images.githubusercontent.com/56163492/204706107-fcceaac5-101e-48eb-9cf3-3828301ff9dc.png)


## Checklist

- [N/A] I have commented my code, particularly in hard-to-understand areas

## How to test

Open Project Settings and click on the dropdown for 'Default Interface Language'. Verify that the 'semantic domains only' language options are no longer in the list.

## qa.languageforge.org testing

Testers should add his/her findings to end of the PR in a comment and include screenshots, files, etc that are beneficial.
